docs: Qms Refactor (#420)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/quality-management/quality_feedback.md

<!-- add-breadcrumbs -->
# Quality Feedback

>  It is the information that a Quality expresses about the satisfaction or dissatisfaction about the products or services.

Reviewer: ```suggestion
**Quality Feedback is the information that a Customer expresses about the satisfaction or dissatisfaction about the products or services you offer.**
```
Quality expresses? 😒

>  The Quality Feedback has to rate different parameters and provide Qualitative Feedbacks using the pre-determined Feedback Templates.

Reviewer: ```suggestion
Quality Feedback has different fields to rate different parameters. You can also provide Qualitative Feedbacks using Quality Feedback Templates. The templates store preset information for you to select in the Quality Feedback.
```

## Quality Feedback Template

>  For creating the Quality Feedback first you must to decide the parameters by creating the Quality Feedback Template. The templates generally contains the title of the template, scope and the different parameter to be entered. The diffrent parameter which can be decided on the basis of different occssions and survey.

Reviewer: ```suggestion
 For creating the Quality Feedback first you must to decide the parameters by creating the Quality Feedback Template. The templates generally contain the name the template, scope and the different parameters to be entered. The different parameters which can be decided on the basis of different occasions and survey.
```
scope how?

> 

Reviewer: To access Quality Feedback Template, go to:
> Home > Quality > Feedback > Quality Feedback Template

> ## Creating a Quality Feedback Template

Reviewer: ```suggestion
## 1. How to create a Quality Feedback Template
```

 <img class="screenshot" alt="Quality mangement" src="{{docs_base_url}}/assets/img/quality-management/template.gif">

>  Steps for creating a Quality Feedback Template

Reviewer: ```suggestion
```
no need to repeat, it's mentioned in the heading

> 

Reviewer: 1. Go to the Quality Feedback Template list, click on New.

>  * Template : The name of the template

Reviewer: ```suggestion
1. Enter a name for the template
```

>  * Parameter : Listing down various processes/parameters to which you want the user to rate and give the feedback.

Reviewer: ```suggestion
1. In the Parameters table, list down the various processes/parameters on which you want the user to rate and give the feedback.
1. Save.
```

## Quality Feedback

>  The Quality can easily give there feedbacks by selecting there template to rate on different parameter and also provide the qualitative feedback on the parameters.

Reviewer: ```suggestion
The Customer can easily give their feedbacks by selecting their template to rate on different parameters and also provide qualitative feedback on these parameters.
```
Quality will give feedback to quality right? nice

>  If the Feedback is not as per expectations, Quality Action may be initialised on the basis of the seriousness of the Feedback.

Reviewer: ```suggestion
If the Quality Feedback ratings are poor or negative, Quality Action may be initialized depending on seriousness of the Quality Feedback.
```

> ## Creating a Quality Feedback

Reviewer: ```suggestion
## 1. How to create a Quality Feedback
```

> 

Reviewer: To access Quality Feedback, go to:
> Home > Quality > Feedback > Quality Feedback

>  <img class="screenshot" alt="Quality mangement" src="{{docs_base_url}}/assets/img/quality-management/feedback.gif">

Reviewer: this GIF name is not present in the PR

>  Steps for creating a Quality Feedback

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
1. Go to the Quality Feedback list, click on New.
```

>  * Type: Select whether Feedback is by a Customer or a User.

Reviewer: ```suggestion
1. Select whether Feedback is by a Customer or a User (ERPNext account User).
```

>  * Name: Name of the Customer or User.

Reviewer: ```suggestion
1. Select the Customer or User.
```

>  * Template : Selecting the template for the Feedback and the parameters are automatically populated for the Feedback.

Reviewer: ```suggestion
1. Selecting a Quality Feedback Template will automatically populate the parameters for the Quality Feedback.
```

> 

Reviewer: 1. The date will be automatically set to the current date.

>  * Feedback :

Reviewer: ```suggestion
1. The Parameters table has the following columns:
```

>     - Parameter : Various Parameters that are pre-defined in the Feedback Template.

Reviewer: ```suggestion
    - Parameter: Various Parameters that are pre-defined in the Quality Feedback Template you create.
```

>     - Rating : Rate a parameter.

Reviewer: ```suggestion
    - Rating: Rating for a parameter from 1 to 5.
```

>     - Feedback : Feedback regarding the parameter.

Reviewer: ```suggestion
    - Feedback: Feedback based on the Rating given.
```
